crates/turborepo-lockfiles/src/npm.rs

```diff
@@ -135,6 +138,16 @@ impl Lockfile for NpmLockfile {
     fn encode(&self) -> Result<Vec<u8>, crate::Error> {
         Ok(serde_json::to_vec_pretty(&self)?)
     }
+
+    fn global_change_key(&self) -> Vec<u8> {
+        let mut buf = vec![b'n', b'p', b'm', 0];
```

Comment:
Is the null byte important?

Reply:
No, I had it do a null byte just to ensure that there are no clashes if the prefixes overlapped.

Reply:
very unlikely situation now that the rest of the data is structured json, but whatever